databases/week1/hyf-homework1.sql

```diff
@@ -0,0 +1,35 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(id) as tasks FROM task;
+
+/*Find out how many tasks in the task table do not have a valid due date*/
+
+SELECT COUNT(id) as tasks FROM task WHERE due_date = NULL;
```

Comment:
You should use IS NULL instead of = NULL. An explanation of the difference can be found here: https://www.sqlservercentral.com/articles/understanding-the-difference-between-is-null-and-null

Reply:
ok.. fixed it.